orchagent/intfsorch.cpp

```diff
@@ -310,38 +347,31 @@ void IntfsOrch::doTask(Consumer &consumer)
         string op = kfvOp(t);
         if (op == SET_COMMAND)
         {
-            if (alias == "lo")
+            if (is_lo)
             {
                 if (!ip_prefix_in_key)
                 {
-                    it = consumer.m_toSync.erase(it);
-                    continue;
-                }
-
-                bool addIp2Me = false;
-                // set request for lo may come after warm start restore.
-                // It is also to prevent dupicate set requests in normal running case.
-                auto it_intfs = m_syncdIntfses.find(alias);
-                if (it_intfs == m_syncdIntfses.end())
-                {
-                    IntfsEntry intfs_entry;
-
-                    intfs_entry.ref_count = 0;
-                    intfs_entry.ip_addresses.insert(ip_prefix);
-                    m_syncdIntfses[alias] = intfs_entry;
-                    addIp2Me = true;
+                    if (m_syncdIntfses.find(alias) == m_syncdIntfses.end())
```

Comment:
Looking at current handling for Loopback interfaces, don't see the need for specific handling and it is same as other interface handling. 
Is there any other reason for doing specific handling for loopback?
If not, can we have common code for all interface types (including loopback)?

Reply:
> Looking at current handling for Loopback interfaces, don't see the need for specific handling and it is same as other interface handling.
> Is there any other reason for doing specific handling for loopback?
> If not, can we have common code for all interface types (including loopback)?

Loopback interfaces don't create router interface in SAI. Loopback interfaces don't exists in PortsOrch. So handling for loopback isn't move into setIntf/removeIntf all the time.